### ISSUE TITLE
AppCue setup for Onboarding video

### ISSUE THREAD

**Issue by veselala** [humanmade/altis-analytics#322]
Onboarding video for the Beta public release should be shown to users, that are entering the the WP dashboard for a first time, after the site was added to our SaaS plugin. 

- [ ] AppCue event should be triggered when the user reaches a page that ends with /wp-admin/ (as on the screenshot below)
- [ ] AppCue event is triggered only once

![image.png](https://images.zenhubusercontent.com/61dc60222e85a543a5e62d05/fdf165e8-f541-401d-ae24-e2576b99aa0a)
![image.png](https://images.zenhubusercontent.com/61dc60222e85a543a5e62d05/e49b313b-1aa9-44a5-83a0-3a32349059f3)

Onboarding video should be provided as a result of #1074

**Comment by noeltock** [comment]
Initial onboarding flow done and deployed. Will create a new issue when I have the opportunity to use the plugin and record it.
